### ISSUE TITLE
I was not able to attach superfamily Curculionoidea in assembly tree

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/backend#1030]
Frontend version: 53d41a0 June 29, 2021 8:17 AMBackend version: 0173dc6 June 28, 2021 12:45 PM
at https://data.catalogueoflife.org/catalogue/3/assembly?datasetKey=1039

I need to establish a new sector in the tree, superfamily Curculionoidea.

I have drug&dropped superfamily Curculionoidea from WTaxa into order Coleoptera. Pushed button "Attach", and nothing happens for a while.

![image](https://user-images.githubusercontent.com/28574252/123855720-d48c3180-d8e5-11eb-901c-2a937b56c0c0.png)

Refreshed page in a browser, pop-up menu disappeared, but superfamily Curculionoidea did not appear in the Tree (even after 30 mins).

**Comment by yroskov** [comment]
It's a blocker for July edition.

**Comment by gdower** [comment]
The frontend is sending an empty request:

![image](https://user-images.githubusercontent.com/8573609/123860524-a14ca100-d8eb-11eb-92ea-3638251c10dc.png)

The browser console output is:

```
Error: Request failed with status code 503
    createError createError.js:16
    settle settle.js:18
    handleLoad xhr.js:59
    dispatchXhrRequest xhr.js:34
    xhrAdapter xhr.js:11
    dispatchRequest dispatchRequest.js:59
    promise callback*request Axios.js:51
    method Axios.js:71
    wrap bind.js:9
    saveSector index.js:176
    getSectorInfo index.js:96
    confirmAttach ColTree.js:875
    action ColTree.js:1327
    onClick PopconfirmMultiOption.js:110
    handleClick button.js:195
    React 12
    unstable_runWithPriority scheduler.production.min.js:19
    React 3
index.js:187:16
```
I don't see a 503 error in the backend logs, so maybe this is a frontend issue.

**Comment by gdower** [comment]
Not sure if it matters or not, but Yuri might have hit the recursive delete button more than once because the frontend did not pop-up a notification that it deleted until a long delay after deleting a really large subtree so it gave the impression that nothing was being done. I see multiple deletes in the logs:

https://logs.gbif.org/app/discover#/context/37c669c0-2a5c-11eb-9ca0-ddc1af98892f/DUAUWXoBBVmQ6cT2p8x-?_g=(filters:!())&_a=(columns:!(level,logger_name,message),filters:!(),predecessorCount:10,sort:!('@timestamp',desc),successorCount:15)

**Comment by gdower** [comment]
After a 6 minute lag the frontend does eventually show an error:

![image](https://user-images.githubusercontent.com/8573609/123865230-44ec8000-d8f1-11eb-9e88-34bbf43cf8df.png)

```
Request failed with status code 503

HTTP method: POST

https://api.catalogueoflife.org/dataset/3/sector

Body:

{"subjectDatasetKey":1039,"datasetKey":"3","mode":"ATTACH","subject":{"id":"Curculionoidea","status":"accepted","rank":"superfamily"},"target":{"id":"57b4cba3-9a7d-493b-b44c-866591658f54","status":"accepted","rank":"order"}}
```

**Comment by gdower** [comment]
```
Error 503 Backend fetch failed

Backend fetch failed
Guru Meditation:

XID: 654611096

Varnish cache server
```

**Comment by mdoering** [comment]
The sector attach request is hanging - maybe indeed because there is a running deletion still that blocks it, I need to check. @thomasstjerne could we show a spinning wheel while the UI tries to do the sector POST so a user knows at least it is trying to do something?

**Comment by mdoering** [comment]
There are many deletions and sector creations piled up in postgres that block each other apparently.
The recursive taxon subtree deletion also has to remove nested sectors. In this case there are 3 that get triggered:

> Delete sector 662 from project 3 and its imports by user yroskov {102}
> Delete sector 512 from project 3 and its imports by user yroskov {102}
> Delete sector 592 from project 3 and its imports by user yroskov {102}

I will try to kill all running sql statements and deletion jobs.

**Comment by mdoering** [comment]
jobs are running again and syncs appear. 
Curculionoidea is attached now and I have started a sync to see if it all works.

@yroskov please check all the things you did yesterday. As I killed some postgres queries we might have missed (parts of) a deletion job.

**Comment by yroskov** [comment]
@mdoering, thanks! Curculionoidea is in place. I have succeeded to add sector for family Brentidae (Brentids). 

However, when I tried to establish sector for subfamily Entiminae (3i Curculio) - drag&drop - system did not react (there is no pop-up menu "Replace", "Union" etc) - it lasted for 10 mins now without any displayed results. Still a problem?

![image](https://user-images.githubusercontent.com/28574252/123965163-cb4ca480-d979-11eb-9bca-6903b054e38b.png)
